lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1276,6 +1077,24 @@ export class UserAgentApplication {
 
   //#region Response
 
+  /**
+   * @hidden
+   * @ignore
+   * Checks if the redirect response is received from the STS. In case of redirect, the url fragment has either id_token, access_token or error.
+   * @param {string} hash - Hash passed from redirect page.
+   * @returns {Boolean} - true if response contains id_token, access_token or error, false otherwise.
+   */
+  urlContainsHash(hash: string): boolean {
+    hash = this.getHash(hash);
```

Comment:
I think the parameter should be called `url`, instead of hash:

```js
urlContainsHash(url: string): boolean {
    const hash = this.getHash(url );
```

Reply:
Or if a hash value if the only value that should be passed to this, maybe change the name to `isStsResponse` or something?

Reply:
It is mostly called hash as we read only the fragment of the STSResponse to populate the hash.

Reply:
Thanks for the feedback guys, I made some changes. I will push and we can review later.